pkgs/development/libraries/science/math/magma/releases.nix

```diff
@@ -0,0 +1,93 @@
+[
```

Comment:
might be worth leaving a comment in this file that the order of this list matters. in particular it should be sorted in ascending version order iiuc